### DISCUSSION TITLE
Resources section of the docs

### DISCUSSION THREAD

**Discussion by ronaldborman** [statelyai/xstate#2500] [General] [answered]
Hi, I have some thoughts on the Resources section of the docs. IMHO articles and videos should be separated because they are consumed differently, at least by me ;-). I also think the different lists should be in reverse chronological order (so the newest items are at the top) by adding the publication or scheduled date. And the Tutorials section could be merged in as well.

Is this something you would be interested in? If so, I'm willing to make a pull request.

**Comment by davidkpiano** [comment, marked as answer]
> Is this something you would be interested in? If so, I'm willing to make a pull request.

Yes please! Would happily accept a pull request for this.
